spec/factories/merchants.rb

```diff
@@ -1,5 +1,16 @@
 FactoryBot.define do
   factory :merchant do
-    name { "MyString" }
+    name { Faker::Company.name }
+    factory :merchant_with_items do
+      transient do # this passes to options below 
+        num { 5 }
```

Comment:
This will limit the merchants items to five and also making sure that there is not just one new merchant to each item. Will there be a time that we need more than five? Will this also support having random days of the week (user story #13)